consensus/src/chained_bft/safety/safety_rules.rs

```diff
@@ -218,13 +218,6 @@ impl SafetyRules {
         None
     }
 
-    /// Return the new state if the voting round was increased, otherwise ignore.  Increasing the
-    /// last vote round is always safe, but can affect liveness and must be increasing
-    /// to protect safety.
-    pub fn increase_last_vote_round(&mut self, round: Round) -> Option<ConsensusState> {
```

Comment:
ah I commented above can we delete this function and see you actually delete it!